.socialgouv/values.project.yaml

```diff
@@ -0,0 +1,10 @@
+global:
+  projectName: standup
+  
+components:
+  app: true
+
+app:
+  envSecretName: standup-sealed-secret
+  envConfigName: standup-configmap
+  appPort: 80
```

Comment:
containerPort ?

Reply:
cela dit tu as raison ça part quand même du port du container au départ et c'est cela qui induit le reste, OK je change en `containerPort`

Reply:
it's done